### PR TITLE
Updated upload process [Enhancement]

### PR DESCRIPTION
**Pull request by akhil-rana**
## Description
- The upload process has been made easy for the end user.
- Number of steps (button clicks) for the end user has been reduced.
- Previously, there was a bug in which the file upload progress was shown incorrectly. The bug is now fixed.
- A user-friendly progress bar with live file upload progress has been added.
- The backend has not been altered.

## Motivation and Context
- Previous upload process seemed to be a little cumbersome for the end user.

## How Has This Been Tested?
- Browser: Firefox (v83), Chromium (v87)
- OS: Windows 10, Arch KDE (Linux-LTS)

## Screenshots:
out

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/loader/chunked_upload.js [hunk 1]
```diff
@@ -79,13 +83,17 @@ async function startUpload(filename) {
 }
 
 function continueUpload(token) {
-  return async function(body) {
-    changeStatus('UPLOAD', 'Uploading chunk at: '+ body.offset +' of size '+ body.chunkSize);
+  return async function(body, file) {
+    let progressValue = Math.floor((body.offset/file.size)*100);
+    $('#upload-progress').css('width', progressValue+'%').attr('aria-valuenow', progressValue).text(progressValue + '%');
+    changeStatus('UPLOAD', 'Uploading chunk at: '+ body.offset/(1024*1024) +'MB of size '+
```
Comment (birm) [inline apps/loader/chunked_upload.js:89]:
```suggestion
    changeStatus('UPLOAD', 'Uploading chunk at: '+ body.offset/(1024*1024) +'MB of total '+
```

This originally referred to the chunk size, which you've correctly identified as useless info to the user. Just want to make it clearer that this new value is the total.

**Review by birm — APPROVED** [review]
Outside of a small wording suggestion, this looks great!!

**Comment by akhil-rana** [comment]
Thanks for the suggestion and review 🙂

**Comment by birm** [comment]
Awesome, thank you @akhil-rana

**Pull request merged**